APIURLS.md

```diff
@@ -0,0 +1,110 @@
+Admin Features:
+    Manage Project:
+        Projects Database:
+            localhost:8080/API/v1/project/id/1
+            [Post, Put, Get, Delete]
+        Create Project:
+            POST localhost:8080/API/v1/projects/id
+        Search Project:
+            GET localhost:8080/API/v1/projects?search=id

```

Comment:
What is the response (in json format)?

Reply:
The response of this GET request should be like this"

{
  "success": true,
  "projects": [
    {
      "id": "12345",
      "name": "exampleName",
      "startDate": "2024-01-01",
      "endDate": "2024-12-31",
      "Assignee": [
        {
          "userId": "67890",
        },
      ]
      "Reporter": [
        {
          "userId": "54321",
        }
      ]
    },
 ]
}